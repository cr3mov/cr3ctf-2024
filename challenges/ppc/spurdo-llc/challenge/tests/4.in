100 182200
3408 3989 1270 527 1616 101 3719 289 2505 3010 860 4292 3804 652 1937 3814 4280 3739 2600 4456 3256 3177 1740 3549 1557 1862 4792 4585 3947 42 4271 792 4931 755 2094 1305 2803 1296 2482 275 4885 2834 1147 4487 4337 4745 3515 1360 2587 4994 915 4293 12 3830 4586 946 2312 610 3826 691 4021 4026 2761 27 2366 2113 1460 2201 585 4258 104 429 4546 4582 1454 3485 656 3367 4823 2754 4321 1769 4086 2150 435 2738 2967 705 1105 371 4357 257 4995 369 3033 2334 2586 4260 2014 1938
