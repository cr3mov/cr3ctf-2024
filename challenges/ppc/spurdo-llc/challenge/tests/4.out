800 800 800 527 800 101 800 289 800 800 800 800 800 652 800 800 800 800 800 800 800 800 800 800 800 800 800 800 800 42 800 792 800 755 800 800 800 800 800 275 800 800 800 800 800 800 800 800 800 800 800 800 12 800 800 800 800 610 800 691 800 800 800 27 800 800 800 800 585 800 104 429 800 800 800 800 656 800 800 800 801 801 801 801 435 801 801 705 801 371 801 257 801 369 801 801 801 801 801 801 